platform/android/export/export.cpp

```diff
@@ -1341,6 +1356,43 @@ class EditorExportPlatformAndroid : public EditorExportPlatform {
 			return str;
 		}
 	}
+
+	// Creates strings.xml files inside the gradle project for different locales.
+	void _create_project_name_strings_files(const Ref<EditorExportPreset> &p_preset) {
+		const char *locales[] = {
+			"ar", "bg", "ca", "cs", "da", "de", "el", "en", "es", "es_ES", "fa", "fi", "fr", "hi",
+			"hr", "hu", "in", "it", "iw", "ja", "ko", "lt", "lv", "nb", "nl", "pl", "pt", "ro",
+			"ru", "sk", "sl", "sr", "sv", "th", "tl", "tr", "uk", "vi", "zh", "zh_HK", "zh_TW"
+		};
+		// TODO: should we use "package/name" or "application/config/name"?
+		String package_name = p_preset->get("package/name");
+		String project_name = get_project_name(package_name).xml_escape(true);
+		size_t num_locales = sizeof(locales) / sizeof(locales[0]);
+
+		// The following 4 lines take care of writing to base strings.xml (no locale)
+		String base_xml_file_path = "res://android/build/res/values/strings.xml";
+		String base_xml_string = FileAccess::get_file_as_string(base_xml_file_path, NULL);
+		base_xml_string = base_xml_string.replace("godot-project-name", project_name);
+		store_string_at_path(base_xml_file_path, base_xml_string);
+
+		for (size_t i = 0; i < num_locales; i++) {
+			String locale = locales[i];
+			String property_name = "application/config/name_" + locale;
+			String hyphenated_locale = locale.replace("_", "-r");
+			String locale_file_path = "res://android/build/res/values-" + hyphenated_locale + "/strings.xml";
+			String replacement_text = "godot-project-name-" + locale;
```

Comment:
@m4gr3d I followed your suggestion and moved the `strings.xml` files to the `app` module. This is the new solution, can you take a look?